example_test.go

```diff
@@ -0,0 +1,77 @@
+package jwt_test
+
+import (
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/cristalhq/jwt"
+)
+
+func Example() {
+	// 1. create a signer
+	key := []byte(`secret`)
+	signer, errSigner := jwt.NewHS256(key)
+	checkErr(errSigner)
+
+	// 2. create a builder based on signer
+	builder := jwt.NewTokenBuilder(signer)
+
+	// 3. create standard claims (you can use your own)
```

Comment:
Maybe: `you can use your own as soon as custom claims implement MarshalBinary interface`.